.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,33 @@
+---
+name: Bug report
+about: Create a report to help us improve
```

Comment:
Create a report to help us improve almost sounds like an incomplete sentence, maybe change this to something like `Create a report to help us improve instructlab`